hudi-common/src/main/java/org/apache/hudi/io/storage/HoodieBaseParquetWriter.java

```diff
@@ -67,6 +89,22 @@ public HoodieBaseParquetWriter(Path file,
     this.recordCountForNextSizeCheck = DEFAULT_MINIMUM_RECORD_COUNT_FOR_CHECK;
   }
 
+  protected void handleParquetBloomFilters(ParquetWriter.Builder parquetWriterbuilder, Configuration hadoopConf) {
+    parquetWriterbuilder.withBloomFilterEnabled(ParquetOutputFormat.getBloomFilterEnabled(hadoopConf));
+    // inspired from https://github.com/apache/parquet-mr/blob/master/parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java#L458-L464
+    hadoopConf.forEach(conf -> {
+      String key = conf.getKey();
+      if (key.startsWith(BLOOM_FILTER_ENABLED)) {
+        String column = key.substring(BLOOM_FILTER_ENABLED.length() + 1, key.length());
+        parquetWriterbuilder.withBloomFilterEnabled(column, Boolean.valueOf(conf.getValue()));
```

Comment:
So you mean `Iceberg` and `DeltaLake` also handle the `BloomFilter` options sepcifically?

Reply:
Thanks for the clarification, can we move the test as a UT in `TestHoodieAvroParquetWriter` instead of a Spark IT test.

Reply:
We cannot afford removing the IT test because eventually we shall make sure the bloom is leveraged by hudi spark datasource reader. 

Also the IT test makes sure the parquet writer leverages bloom.

The IT is inspired from spark test https://github.com/apache/spark/blob/a72d1aa4910c67c3b8b4dd78997818ef6a601875/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala#L1977

Reply:
The writer should be shared by all the engines, not just Spark, so it is necessary to add a basic UT for it.